contracts/plugins/assets/morpho-aave/IMorpho.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: BlueOak-1.0.0
+pragma solidity 0.8.19;
+
+import { IERC20Metadata, RewardableERC4626Vault } from "../erc20/RewardableERC4626Vault.sol";
+
+interface IMorpho {
+    function supply(address _poolToken, uint256 _amount) external;
+
+    function withdraw(address _poolToken, uint256 _amount) external;
+}
+
+interface IRewardsDistributor {
+    function claim(
+        address _account,
+        uint256 _claimable,
+        bytes32[] calldata _proof
+    ) external;
+}
+
+interface IUsersLens {
+    function getCurrentSupplyBalanceInOf(address _poolToken, address _user)
+        external
+        view
+        returns (
+            uint256 balanceInP2P,
+            uint256 balanceOnPool,
+            uint256 totalBalance
+        );
+}
+
+abstract contract IMorphoTokenisedDeposit is RewardableERC4626Vault {
```

Comment:
Is there a reason to prefer this approach, as opposed to making the interface _just_ the `getExchangeRate()` function? It's also not an interface, despite being named as if it is. 

It would then be `is RewardableERC4626Vault, IMorphoTokenisedDeposit` when it comes to usage in the plugin, which is how we've done it elsewhere.